project 3/CODE/style.css

```
@@ -0,0 +1,123 @@
+body {
+    background: linear-gradient(to bottom, hsla(281, 100%, 25%, 1) 0%, hsla(22, 100%, 63%, 1) 51%, hsla(21, 97%, 63%, 1) 52%, hsla(281, 100%, 25%, 0.97) 100%);
```

Since this is a background color for the whole site, it probably doesn't need transparency and could use HSL:
```suggestion
    background: linear-gradient(to bottom, hsl(281, 100%, 25%) 0%, hsl(22, 100%, 63%) 51%, hsl(21, 97%, 63%) 52%, hsl(281, 100%, 25%) 100%);
```